cachito/jenkins-scripts/fetch-failed-request-logs.py

```diff
@@ -0,0 +1,195 @@
+#!python
+
+# Fetch Failed Cachito Jobs.
```

Comment:
I am thinking we could have a simple example here showing how the script can be run (e.g. which parameters are mandatory).

Reply:
Simple example added to comments.  Epilog added describing either date or yesterday required.  --server marked required.

Reply:
date / yesterday marked mutually exclusive but required.